CMakeLists.txt

```diff
@@ -2,16 +2,25 @@ cmake_minimum_required(VERSION 2.8.3)
 project(class_loader)
 
 find_package(Boost REQUIRED COMPONENTS thread system)
-find_package(catkin REQUIRED cmake_modules)
+find_package(catkin QUIET COMPONENTS cmake_modules)
```

Comment:
Instead of making this decision quietly it should print a message at least in the case that catkin is not being used.

Also consider the case where `catkin` is available but not `cmake_modules`...